include/maliput_sparse/geometry/lane_geometry.h

```diff
@@ -149,6 +149,11 @@ class LaneGeometry {
   /// @param p P parameter of the centerline.
   double FromCenterPToLateralP(const LineStringType& line_string_type, double p) const;
 
+  /// Computes the inertial position in a lateral boundary for a given @p p .
+  /// @param line_string_type The type of LineString to convert p coordinate to.
+  /// @param p P parameter of the centerline.
+  maliput::math::Vector3 ToLateralPos(const LineStringType& line_string_type, double p) const;
```

Comment:
I thought about the same when I added the method. At first, it isn't necessary, there is no use outside this class, however, it could be useful at some point if we happen to create a utility method(outside this class) that uses the geometry of the lane in order to do some geometrical computation.

Reply:
I suggest you keep it private until you need it public then.